docs/source/user_guide/data_checks.ipynb

```diff
@@ -102,14 +254,14 @@
    },
    "outputs": [],
    "source": [
-    "data = np.random.randn(100, 100)\n",
+    "data = np.tile(np.arange(10) * 0.01, (100, 10))\n",
```

Comment:
For OutliersDataCheck, we now don't use Isolation Forest, but rather just classic stats to find outliers in the cols.